inbm/Changelog.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## NEXT - ?.?.? - ?
 
+### Fixed
+ - RTC 538468 - paho-mqtt upgrade broke cloudadapter's mqtt connections. Fixed proxy setting code to not override all sockets with proxy as paho-mqtt 1.6.0 relies on listening/connecting to localhost to set up sockets, and this doesn't work with a global proxy on all sockets.
```

Comment:
<h3>$\color{#FFD700}Coding Style: CodeDocumentation$</h3>

Ensure that the Changelog entry for the bug fix is clear and provides enough context for users to understand the impact and resolution.

Reply:
Unsure why this is being flagged as possible unclear.